data/capture_file.go

```diff
@@ -14,19 +15,20 @@ import (
 	"github.com/matttproud/golang_protobuf_extensions/pbutil"
 	"github.com/pkg/errors"
 	v1 "go.viam.com/api/app/datasync/v1"
+	"google.golang.org/protobuf/types/known/anypb"
 
-	"go.viam.com/rdk/protoutils"
 	"go.viam.com/rdk/resource"
 	"go.viam.com/rdk/utils"
 )
 
 // TODO Data-343: Reorganize this into a more standard interface/package, and add tests.
 
-// FileExt defines the file extension for Viam data capture files.
+// CompletedCaptureFileExt defines the file extension for Viam data capture files
+// which are no longer being written to.
 const (
-	InProgressFileExt = ".prog"
-	FileExt           = ".capture"
-	readImage         = "ReadImage"
+	InProgressCaptureFileExt = ".prog"
+	CompletedCaptureFileExt  = ".capture"
```

Comment:
renamed to make it clear that this is the extension for completed data capture files